### DISCUSSION TITLE
Control or Command click on a link to open a new tab but not leave the current page right away

### DISCUSSION THREAD

**Discussion by dandline9** [will-stone/browserosaurus#630] [Ideas]
[Looks like it will be a nice addition to my utilities collection, thanks for creating it!]

I assume this has been requested already, but don't see it and didn't find it on a search.

An example of what I'm requesting: I'm often clicking on multiple links on a page (checking out various referenced resources) and it's inconvenient to have to leave the page each time; I'd rather open all the pages I'm interested in while staying on the bibliography page and then check out what I've clicked on when I'm done.

Could browserosaurus pass on a CTRL-click (or Mac CMD-click) to the operating system rather than just a regular click?

Thanks!

**Comment by will-stone** [comment]
Hi. Holding alt/option when you select the browser should open the browser tab in the background, is that what you’re looking for?

**Reply by dandline9** [reply]
Thanks for the quick and super helpful reply! Yes, this is the feature I am referring too, didn't have the right vocabulary. 

It's unfortunately not working 100% for me:

   1. My current default browser is Firefox (Mac), so I guess that is why that browser is Browsersaurus list item 1.
   2. When I do an option click to choose Firefox, or Safari (list item 4) a tab does open in the background, the desired behavior... but when I select Chrome (list item 2) it doesn't matter, the new tab opens in the foreground rather than background (I'm taken off the page, not the desired effect).

Is this expected behavior? I'm on the latest Ventura on Mac M2 Mini.

Quick additional questions, unrelated I think:

  4. top of Browsersaurus list -- does this have to be the default browser, or could it be the last browser used (either by default or user configured)? The latter option seems like would be the most likely to let me click on the default list item without having to do any mouse movement before clicking. The default browser may be the most common option, but when working on a project I might want all the links to open in a non-default browser and it seems way more efficient for it to default to the last selected, which would cover both normal (default browser) situations but also special situations like projects where the non-default browser is desired.

  5. Microsoft Teams (which I never use) is showing up as a browser in list item 3... is this expected behavior?
  
Thanks!!

**Reply by dandline9** [reply]
Can anyone address whether the Alt/Option click for background should work on Chrome (as it does for Firefox and Safari)? Or does it depend (somehow) on my configuration, e.g. what is default browser, or does it only work with the default browser, or ??

Thanks!
